hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/routes/HandleResponseRoute.java

```diff
@@ -0,0 +1,33 @@
+package ca.bc.gov.hlth.hnsecure.routes;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.builder.RouteBuilder;
+
+import ca.bc.gov.hlth.hnsecure.audit.AuditSetupProcessor;
+import ca.bc.gov.hlth.hnsecure.audit.entities.TransactionEventType;
+import ca.bc.gov.hlth.hnsecure.filedrops.ResponseFileDropGenerater;
+import ca.bc.gov.hlth.hnsecure.json.Base64Encoder;
+import ca.bc.gov.hlth.hnsecure.json.fhir.ProcessV2ToJson;
+import ca.bc.gov.hlth.hnsecure.parsing.Util;
+
+public class HandleResponseRoute extends RouteBuilder {
+
+	@Override
+	public void configure() throws Exception {
+		from("direct:handleResponse").routeId("handle-response-route")
+			// create filedrops if enabled
+	    	.choice().when(exchangeProperty(Util.PROPERTY_IS_FILE_DROPS_ENABLED).isEqualToIgnoreCase(Boolean.TRUE))
+	    		.bean(ResponseFileDropGenerater.class).id("ResponseFileDropGenerater")
```

Comment:
it looks like here we call the filedrop directly, and elsewhere we wiretap it to "direct:start" (poorly named route)
Can we use the same approach for both?

So something like 
to("direct:requestFiledrop")
to("direct:responseFiledrop")

this could maybe be a later bug fix/refactor

Reply:
I'm having a look. If it's simple (no side effects) I'll incorporate it here.

Reply:
This was completed. It was straightforward.